usmqe_tests/api/gluster/test_crud_gluster_volume.py

```diff
@@ -0,0 +1,301 @@
+#         ===========
+#
+#         Try to stop volume with given name and cluster id via API.
+#
+#         .. test_step:: 1
+#
+#                 Connect to Tendrl API via POST request to ``APIURL/:cluster_id/GlusterStopVolume``
+#                 Where cluster_id is set to predefined value.
+#
+#         .. test_result:: 1
+#
+#                 Server should return response in JSON format:
+#
+#                 Return code should be **202** with data ``{"message": "Accepted"}``.
+#                 job should finish.
+#                 """
+#
+#     api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+#     volume_id = gluster.GlusterVolume(VOLUME_NAME).get_volume_id()
+#     volume_data = {
+#         "Volume.vol_id": volume_id,
+#         "Volume.volname": VOLUME_NAME,
+#
+#     }
+#
+#     job_id = api.stop_volume(cluster_reuse["cluster_id"], volume_data)["job_id"]
+#     api.wait_for_job_status(job_id)
+#     volume = gluster.GlusterVolume(VOLUME_NAME)
+#     volume.check_status("Stopped")
+#     status = api.get_volume_list(cluster_reuse["cluster_id"])[0][volume_id]["status"]
+#     pytest.check(
+#         status == "Stopped",
+#         "Status from API is {}, should be 'Stopped'".format(status),
+#         issue="https://github.com/Tendrl/tendrl-api/issues/56")
+#
+#
+# @pytest.mark.gluster_volume_crud
+# def test_start_volume_valid(
+#         cluster_reuse,
+#         valid_session_credentials):
+#     """@pylatest api/gluster.start_volume_valid
+#         API-gluster: start_volume
+#         ******************************
+#
+#         .. test_metadata:: author fbalak@redhat.com
+#
+#         Description
+#         ===========
+#
+#         Try to start volume with given name and cluster id via API.
+#
+#         .. test_step:: 1
+#
+#                 Connect to Tendrl API via POST request to
+#                 ``APIURL/:cluster_id/GlusterStartVolume``
+#                 Where cluster_id is set to predefined value.
+#
+#         .. test_result:: 1
+#
+#                 Server should return response in JSON format:
+#
+#                 Return code should be **202** with data ``{"message": "Accepted"}``.
+#                 job should finish.
+#                 """
+#
+#     api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+#     volume_id = gluster.GlusterVolume(VOLUME_NAME).get_volume_id()
+#     volume_data = {
+#         "Volume.vol_id": volume_id,
+#         "Volume.volname": VOLUME_NAME,
+#     }
+#
+#     job_id = api.start_volume(cluster_reuse["cluster_id"], volume_data)["job_id"]
+#     api.wait_for_job_status(job_id)
+#     volume = gluster.GlusterVolume(VOLUME_NAME)
+#     volume.check_status("Started")
+#     status = api.get_volume_list(cluster_reuse["cluster_id"])[0][volume_id]["status"]
+#     pytest.check(
+#         status == "Started",
+#         "Status from API is {}, should be 'Started'".format(status),
+#         issue="https://github.com/Tendrl/tendrl-api/issues/55")
+#
```

Comment:
I didn't review the commented code. Just one note did we really want to have so much commented code here?

Reply:
I've left commented code here because it includes valid testcases which should be rewritten with new fixtures.